IFC4x3/PropertyEnumerations/v/PEnum_VerticalOffsetDirection.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="PEnum_VerticalOffsetDirection" UniqueId="80bbc461-9df0-4725-b843-66a02d8c673d">
+	<Documentation>Gives the options for positive vertical offset direction. The values are _LRM_UP_ and _LRM_DOWN_. This vertical offset direction is as viewed from along the linear element facing in the direction of increasing measure.
```

Comment:
How does an increasing measure on the linear element influence up and down definitions? It does not help at all actually without an initial context. That is not available in psets.